src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
```

Comment:
```suggestion
  readonly shape = 'triangle';
```
Prop shape is readonly for each figure.